### PR TITLE
Updating In Progress with story point guidance and concrete steps

### PR DESCRIPTION
**Pull request by mdubya66**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -145,6 +145,24 @@ members of the team; each team member should be working on one thing at a time.
 This pipeline is a good candidate for WIP (work-in-progress) limits. WIP limits help ensure your work flows smoothly,
 and help bring to light any blockers or bottlenecks. Adjust WIP limits according to the size of your team.
 
+To move an issue into the In Progress swim lane several steps must be taken.
+
+1. Assign yourself as the owner.
+2. Ensure the milestone is set (if there is one) and also review the labels to ensure they accurately reflect the issue.
+3. Assign an estimateed level of effort. See the below table for guidance for effort mapping.
```
Comment (dougm) [inline CONTRIBUTING.md]: typo: estimateed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -145,6 +145,24 @@ members of the team; each team member should be working on one thing at a time.
...
+
+After an issue is In Progress it is the best practice to regularily update the issue with current progress and any discussions that may occur via the various collaboration tools used. An issue that is in progress should not go more than 2 days without updates.
```
Comment (dougm) [inline CONTRIBUTING.md]: typo: regularily

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

**Comment by hmahmood** [comment]
LGTM

**Pull request merged**